### ISSUE TITLE
Snowflake- Unparsable section due to 'CREATE SECURE VIEW...'

### ISSUE THREAD

**Issue by RubidiumBlade** [sqlfluff/sqlfluff#816]
-- If this is a parsing or linting issue, please include a minimal SQL example which reproduces the issue, along with the `sqlfluff parse` output, `sqlfluff lint` output and `sqlfluff fix` output when relevant. Finally, please add any labels that are related to the issue before submitting so that it can be addressed easily! --

## Expected Behaviour
Snowflake supports using the `SECURE` option when creating a view, as well as adding a comment. Both of these create an unparsable section.
## Observed Behaviour
These keywords create an uparsable section, even with the Snowflake Dialect selected.
## Steps to Reproduce
`CREATE SECURE VIEW myview AS SELECT col1, col2 FROM mytable;`
`CREATE VIEW myview COMMENT='Test view' AS SELECT col1, col2 FROM mytable;`
`CREATE SECURE VIEW myview COMMENT='Test view' AS SELECT col1, col2 FROM mytable;`
`python -m sqlfluff parse file3.sql --dialect snowflake`:
```sql
== [file3.sql] FAIL
L:   1 | P:   1 |  PRS | Found unparsable section: 'CREATE OR REPLACE SECURE VIEW
                       | myview COM...'
L:   1 | P:  98 | L009 | Files must end with a trailing newline.
```

`python -m sqlfluff parse file3.sql --dialect snowflake`:
```sql
[0](1, 1, 1)        |file:
[0](1, 1, 1)        |    statement:
[0](1, 1, 1)        |        unparsable:                                           !! Expected: 'statement'
[0](1, 1, 1)        |            raw:                                              'CREATE'
[6](1, 1, 7)        |            whitespace:                                       ' '
[7](1, 1, 8)        |            raw:                                              'SECURE'
[13](1, 1, 14)      |            whitespace:                                       ' '
[14](1, 1, 15)      |            raw:                                              'VIEW'
[18](1, 1, 19)      |            whitespace:                                       ' '
[19](1, 1, 20)      |            raw:                                              'myview'
[25](1, 1, 26)      |            whitespace:                                       ' '
[26](1, 1, 27)      |            raw:                                              'COMMENT'
[33](1, 1, 34)      |            raw:                                              '='
[34](1, 1, 35)      |            raw:                                              "'Test view'"
[45](1, 1, 46)      |            whitespace:                                       ' '
[46](1, 1, 47)      |            raw:                                              'AS'
[48](1, 1, 49)      |            whitespace:                                       ' '
[49](1, 1, 50)      |            raw:                                              'SELECT'
[55](1, 1, 56)      |            whitespace:                                       ' '
[56](1, 1, 57)      |            raw:                                              'col1'
[60](1, 1, 61)      |            comma:                                            ','
[61](1, 1, 62)      |            whitespace:                                       ' '
[62](1, 1, 63)      |            raw:                                              'col2'
[66](1, 1, 67)      |            whitespace:                                       ' '
[67](1, 1, 68)      |            raw:                                              'FROM'
[71](1, 1, 72)      |            whitespace:                                       ' '
[72](1, 1, 73)      |            raw:                                              'mytable'
[79](1, 1, 80)      |    statement_terminator:                                     ';'

==== parsing violations ====
L:   1 | P:   1 |  PRS | Found unparsable section: "CREATE SECURE VIEW myview COMMENT='Test
                       | ..."
```

`python -m sqlfluff lint file3.sql --dialect snowflake`:
```sql
== [file3.sql] FAIL
L:   1 | P:   1 |  PRS | Found unparsable section: "CREATE SECURE VIEW myview
                       | COMMENT='Test ..."
L:   1 | P:  80 | L009 | Files must end with a trailing newline.
```

## Version
`sqlfluff --version`: `__main__.py, version 0.4.0`
python version 3.8.5

## Configuration
No config.
